agreements.md

```diff
@@ -45,7 +45,7 @@ This is an agreed upon gentle reminder, that carries no criticism or hard feelin
 
 2. Other team members may react to the message in the following ways:
 
-- 👎 (`:thumbsdown:`) downvote emoji – “I am busy and cannot help you right now”,
+- ⌛ (`:hourglass:`) – “I am busy and cannot help you right now – but I'll come back as soon as possible”,
```

Comment:
```suggestion
- 👋 (`:wave:`) – “I  cannot help you right now”,
```
I would keep it vague to make it universal
also :hourglass: IMO can be misinterpreted like you are supposed to wait for something, but you really shouldn't be waiting